griptape/drivers/sql/amazon_redshift_sql_driver.py

```diff
@@ -0,0 +1,106 @@
+import time
+from typing import Optional
+import boto3
+from griptape.drivers import BaseSqlDriver
+from attr import define, field
+
+
+@define
+class AmazonRedshiftSqlDriver(BaseSqlDriver):
+    database: str = field(kw_only=True)
+    cluster_identifier: str = field(default=None, kw_only=True)
+    workgroup_name: str = field(default=None, kw_only=True)
+    db_user: str = field(default=None, kw_only=True)
+    secret_arn: str = field(default=None, kw_only=True)
+    region_name: str = field(default="us-east-1", kw_only=True)
+    wait_sec: float = field(default=0.3, kw_only=True)
+    client = field(default=None, kw_only=True)
+
+    def __attrs_post_init__(self):
+        if self.client == None:
+            session = boto3.Session(region_name=self.region_name)
+            self.client = session.client("redshift-data")
+
+    @staticmethod
+    def _process_rows_from_records(records) -> list[list]:
+        rows = []
+        for r in records:
+            tmp = [c[list(c.keys())[0]] for c in r]
+            rows.append(tmp)
```

Comment:
Just make stuff like this a single statement:

```python
rows.append(
    [c[list(c.keys())[0]] for c in r]
)
```

or better yet replace the whole thing with a one-liner:

```python
return [c[list(c.keys())[0]] for r in records for c in r]
```

Reply:
Cool, I'll update this to the one-liner 😃